docs/rules/no-case-declarations.md

```diff
@@ -39,7 +39,11 @@ Examples of **correct** code for this rule:
 /*eslint no-case-declarations: "error"*/
 /*eslint-env es6*/
 
+/* Declarations outside switch-statements are valid */
```

Comment:
By looking at other example code I think it is more common to use `//` comments.